include/psa/crypto_values.h

```diff
@@ -625,6 +635,13 @@
 #define PSA_ALG_IS_VENDOR_DEFINED(alg)                                  \
     (((alg) & PSA_ALG_VENDOR_FLAG) != 0)
 
+/** Whether an algorithm is a policy (collection of algorithms).
+ *
+ * See also #PSA_ALG_POLICY_FLAG.
```

Comment:
`PSA_ALG_POLICY_FLAG` is an internal constant and should not be referenced from the Doxygen documentation.

Reply:
N/A now that the policy bit is removed.